components/sp-solutions/org.wso2.carbon.sp.solutions.is.analytics/widgets/IsAnalyticsSummary/src/IsAnalyticsSummary.jsx

```diff
@@ -0,0 +1,299 @@
+/*
+ *  Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  WSO2 Inc. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ *
+ */
+
+import Widget from '@wso2-dashboards/widget';
+import VizG from 'react-vizgrammar';
+import {MuiThemeProvider, createMuiTheme} from '@material-ui/core/styles';
+import _ from 'lodash';
+import Button from '@material-ui/core/Button';
+
+const widgetTexts = {
+    Overall: {
+        bodyText: 'Analyze overall login attempts made via WSO2 Identity Server. This includes information about overall flows of authentication took place through Identity Server. A collection of authentication steps is considered as an overall attempt',
```

Comment:
Line length exceeds here.

Reply:
Isn't it okay to exceed the line length for large strings ?